[0m09:34:06.715274 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017418CBE470>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017419F75DE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017419F76E00>]}


============================== 09:34:06.723271 | 2e559cc3-f09e-4f32-a342-ffe2fad4d1e8 ==============================
[0m09:34:06.723271 [info ] [MainThread]: Running with dbt=1.10.4
[0m09:34:06.726271 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\metya\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt init telegram_analytics', 'send_anonymous_usage_stats': 'True'}
[0m09:34:06.919991 [info ] [MainThread]: Creating dbt configuration folder at C:\Users\metya\.dbt
[0m09:34:06.935413 [debug] [MainThread]: Starter project path: C:\Users\metya\anaconda3\envs\telegram-health-analysis\lib\site-packages\dbt\include\starter_project
[0m09:34:06.969296 [info ] [MainThread]: 
Your new dbt project "telegram_analytics" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m09:34:06.971304 [info ] [MainThread]: Setting up your profile.
[0m09:39:12.753806 [info ] [MainThread]: Profile telegram_analytics written to C:\Users\metya\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m09:39:12.759808 [debug] [MainThread]: Command `dbt init` succeeded at 09:39:12.758814 after 306.30 seconds
[0m09:39:12.762813 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017418CBE470>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017419FCC580>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001741A0E6140>]}
[0m09:39:12.765812 [debug] [MainThread]: Flushing usage events
[0m09:39:14.744816 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m09:39:35.489123 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000278AE47A3E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000278AF73C160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000278AF73ED40>]}


============================== 09:39:35.498119 | 4b85b1e4-c3d8-4c25-af99-e72208144145 ==============================
[0m09:39:35.498119 [info ] [MainThread]: Running with dbt=1.10.4
[0m09:39:35.500129 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': 'C:\\Users\\metya\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'introspect': 'True', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m09:39:35.548010 [info ] [MainThread]: dbt version: 1.10.4
[0m09:39:35.550007 [info ] [MainThread]: python version: 3.10.18
[0m09:39:35.552006 [info ] [MainThread]: python path: C:\Users\metya\anaconda3\envs\telegram-health-analysis\python.exe
[0m09:39:35.553009 [info ] [MainThread]: os info: Windows-10-10.0.26100-SP0
[0m09:39:35.667001 [info ] [MainThread]: Using profiles dir at C:\Users\metya\.dbt
[0m09:39:35.669009 [info ] [MainThread]: Using profiles.yml file at C:\Users\metya\.dbt\profiles.yml
[0m09:39:35.671008 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\metya\Desktop\telegram-health-analytics\dbt_project.yml
[0m09:39:35.684002 [info ] [MainThread]: adapter type: postgres
[0m09:39:35.686009 [info ] [MainThread]: adapter version: 1.9.0
[0m09:39:35.688013 [info ] [MainThread]: Configuration:
[0m09:39:35.689013 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m09:39:35.690011 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m09:39:35.692012 [info ] [MainThread]: Required dependencies:
[0m09:39:35.694014 [debug] [MainThread]: Executing "git --help"
[0m09:39:35.826914 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--no-lazy-fetch]\n           [--no-optional-locks] [--no-advice] [--bare] [--git-dir=<path>]\n           [--work-tree=<path>] [--namespace=<name>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone      Clone a repository into a new directory\n   init       Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add        Add file contents to the index\n   mv         Move or rename a file, a directory, or a symlink\n   restore    Restore working tree files\n   rm         Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect     Use binary search to find the commit that introduced a bug\n   diff       Show changes between commits, commit and working tree, etc\n   grep       Print lines matching a pattern\n   log        Show commit logs\n   show       Show various types of objects\n   status     Show the working tree status\n\ngrow, mark and tweak your common history\n   backfill   Download missing objects in a partial clone\n   branch     List, create, or delete branches\n   commit     Record changes to the repository\n   merge      Join two or more development histories together\n   rebase     Reapply commits on top of another base tip\n   reset      Reset current HEAD to the specified state\n   switch     Switch branches\n   tag        Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch      Download objects and refs from another repository\n   pull       Fetch from and integrate with another repository or a local branch\n   push       Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m09:39:35.827913 [debug] [MainThread]: STDERR: "b''"
[0m09:39:35.829917 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m09:39:35.831932 [info ] [MainThread]: Connection:
[0m09:39:35.833923 [info ] [MainThread]:   host: localhost
[0m09:39:35.834923 [info ] [MainThread]:   port: 5432
[0m09:39:35.836920 [info ] [MainThread]:   user: postgres
[0m09:39:35.838927 [info ] [MainThread]:   database: telegram_db
[0m09:39:35.840923 [info ] [MainThread]:   schema: raw
[0m09:39:35.842923 [info ] [MainThread]:   connect_timeout: 10
[0m09:39:35.844924 [info ] [MainThread]:   role: None
[0m09:39:35.846933 [info ] [MainThread]:   search_path: None
[0m09:39:35.848925 [info ] [MainThread]:   keepalives_idle: 0
[0m09:39:35.849924 [info ] [MainThread]:   sslmode: None
[0m09:39:35.851918 [info ] [MainThread]:   sslcert: None
[0m09:39:35.852915 [info ] [MainThread]:   sslkey: None
[0m09:39:35.854922 [info ] [MainThread]:   sslrootcert: None
[0m09:39:35.855918 [info ] [MainThread]:   application_name: dbt
[0m09:39:35.856924 [info ] [MainThread]:   retries: 1
[0m09:39:35.859922 [info ] [MainThread]: Registered adapter: postgres=1.9.0
[0m09:39:36.456144 [debug] [MainThread]: Acquiring new postgres connection 'debug'
[0m09:39:36.706468 [debug] [MainThread]: Using postgres connection "debug"
[0m09:39:36.707468 [debug] [MainThread]: On debug: select 1 as id
[0m09:39:36.708470 [debug] [MainThread]: Opening a new connection, currently in state init
[0m09:39:36.893575 [debug] [MainThread]: Postgres adapter: Got a retryable error when attempting to open a postgres connection.
1 attempts remaining. Retrying in 0 seconds.
Error:
connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "postgres"

[0m09:39:36.952534 [debug] [MainThread]: Postgres adapter: Error running SQL: select 1 as id
[0m09:39:36.954532 [debug] [MainThread]: Postgres adapter: Rolling back transaction.
[0m09:39:36.956544 [debug] [MainThread]: On debug: No close available on handle
[0m09:39:36.958533 [info ] [MainThread]:   Connection test: [[31mERROR[0m]

[0m09:39:36.960534 [info ] [MainThread]: [31m2 checks failed:[0m
[0m09:39:36.962537 [info ] [MainThread]: Project loading failed for the following reason:
 project path <C:\Users\metya\Desktop\telegram-health-analytics\dbt_project.yml> not found

[0m09:39:36.964532 [info ] [MainThread]: dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "postgres"
  

Check your database credentials and try again. For more information, visit:
https://docs.getdbt.com/docs/configure-your-profile


[0m09:39:36.967538 [debug] [MainThread]: Command `dbt debug` failed at 09:39:36.967538 after 1.71 seconds
[0m09:39:36.969537 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m09:39:36.971536 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000278AE47A3E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000278AF8EB820>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000278AF8EAF50>]}
[0m09:39:36.972535 [debug] [MainThread]: Flushing usage events
[0m09:39:38.682559 [debug] [MainThread]: An error was encountered while trying to flush usage events
